analytics_data_api/v0/tests/views/test_course_summaries.py

```diff
@@ -162,3 +167,15 @@ def test_no_matching_courses(self):
     def test_bad_course_id(self, course_ids):
         response = self.authenticated_get(self.path(course_ids=course_ids))
         self.verify_bad_course_id(response)
+
+    def test_collapse_upcoming(self):
+        self.generate_data(availability='Starting Soon')
+        self.generate_data(course_ids=['foo/bar/baz'], availability='Upcoming')
```

Comment:
I'm not sure that this tests that the summation is happening because the course ID used here is different than the one above.  Maybe I'm missing something?